docker-compose.yml

```diff
@@ -31,22 +31,8 @@ services:
       - PCS_WEBUI_AUTH_AAD_AUTHORITY
       - PCS_WEBUI_AUTH_AAD_TENANT
       - PCS_CORS_WHITELIST
-  onboardingservice:
-    image: ${REPOSITORY:-localhost:5000/azure-iiot-}opc-onboarding-service:${IMAGE_VERSION:-latest}
-    build: 
-      context: ./services
-      dockerfile: ./src/Microsoft.Azure.IIoT.Services.OpcUa.Onboarding/docker/Dockerfile
-    environment:
-      - PCS_KEYVAULT_URL
-      - PCS_SERVICEBUS_CONNSTRING
-      - PCS_IOTHUB_CONNSTRING
-      - PCS_IOTHUBREACT_HUB_ENDPOINT
-      - PCS_IOTHUBREACT_HUB_PARTITIONS
-      - PCS_IOTHUBREACT_HUB_NAME
-      - PCS_IOTHUBREACT_HUB_CONSUMERGROUP
-      - PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT
-      - PCS_IOTHUBREACT_AZUREBLOB_KEY
-      - PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX
+      - PCS_APPINSIGHTS_NAME
+      - PCS_APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
I do not see any changes regarding  `PCS_APPINSIGHTS_NAME` and `PCS_APPINSIGHTS_INSTRUMENTATIONKEY` variables outside of `docker-compose.yml` files. Are the variables dumped to `.env` file by deploy script ?

Reply:
Correct.  These are dumped into .env by setup.sh which runs inside on creation of the VM.  And we forgot to update the docker-compose.yml when it was added recently to make use of these.